e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh

```diff
@@ -312,7 +318,8 @@ sp_adf_pass=$(echo "$sp_adf_out" | jq -r '.password')
 sp_adf_tenant=$(echo "$sp_adf_out" | jq -r '.tenant')
 
 # Validate the password
-sp_adf_pass=$(validate_password "$sp_adf_pass")
+sp_adf_pass=$(validate_password "$sp_stor_pass" "$sp_stor_id" )
```

Comment:
you are using the wrong variables here, you are still referencing to the storage variables, you should refer to the ADF variables instead